### PR TITLE
[@mantine/colors-generator] Dependency: upgrade `chroma-js`

### PR DESCRIPTION
**Pull request by GZTimeWalker**
Upgrade `chroma-js` to a newer version, related functions are tested and work fine.

Tip: There seem to be some issues with v2.6.0 that are not working as expected.

### CODE REVIEW

File: packages/@mantine/colors-generator/package.json [hunk 1]
```diff
@@ -37,6 +37,6 @@
     "directory": "packages/@mantine/colors-generator"
   },
   "peerDependencies": {
-    "chroma-js": "^2.4.2"
```
[the remark this answers is not in the document]
Comment (GZTimeWalker) [reply packages/@mantine/colors-generator/package.json:40]: maybe we should ban some problematic versions like v2.6.0?

which will cause `unknown format: #592121`

[the remark this answers is not in the document]
Comment (rtivital) [reply packages/@mantine/colors-generator/package.json:40]: Nah, `>=2.4.2` is fine, most usage is still expected to be the latest version of chroma-js.

**Comment by GZTimeWalker** [comment]
Done.

**Comment by rtivital** [comment]
Thanks!

**Pull request merged**